src/components/profile/ProfileTeamLocation.vue

```diff
@@ -58,18 +96,102 @@ export default {
     locationSearchString() {
       return 'officeLocation:"' + this.officeLocation + '"'; // eslint-disable-line
     },
+    hasBrowserTimezone() {
+      const profileDate = getFormattedDateWithTimezone(
+        this.localtime,
+        this.timezone,
+      );
+      const browserTimezone = getBrowserTimezone();
+      if (browserTimezone && browserTimezone !== this.currentTimezone) {
+        const currentBrowserDate = getFormattedDateWithTimezone(
+          this.localtime,
+          browserTimezone,
+        );
+        const browserHoursDiff = getHoursDiff(profileDate, currentBrowserDate);
+        return browserHoursDiff !== null && browserHoursDiff !== 0;
+      }
+      return false;
+    },
+    hasTimezoneOffset() {
+      return this.timezoneDiff !== '' && this.timezoneDiff !== null;
+    },
     timezoneWithTime() {
+      // Return final string
       if (this.timezone) {
-        return `${this.localtime} local time (${this.getTimezoneName(
+        return `${this.getLocaltime()} local time (${getTimezoneName(
           this.timezone,
         )})`;
       }
       return null;
     },
+    timezoneDiff() {
+      if (!this.timezone || !this.currentTimezone) {
+        return null;
+      }
+
+      // Get viewed profile timezone
+      const profileDate = getFormattedDateWithTimezone(
+        this.localtime,
+        this.timezone,
+      );
+
+      // Get logged in profile timezone
+      const currentLocalDate = getFormattedDateWithTimezone(
+        this.localtime,
+        this.currentTimezone,
+      );
+
+      // Get browser timezone
+      const browserTimezone = getBrowserTimezone();
+      let currentBrowserDate = null;
+      let printedBrowserOffset = '';
+      let browserHoursDiff = null;
+      // Build browser timezone string
+      if (browserTimezone && browserTimezone !== this.currentTimezone) {
+        currentBrowserDate = getFormattedDateWithTimezone(
+          this.localtime,
+          browserTimezone,
+        );
+        browserHoursDiff = getHoursDiff(profileDate, currentBrowserDate);
+        if (browserHoursDiff !== null && browserHoursDiff !== 0) {
+          printedBrowserOffset = `${decimalToHours(
+            browserHoursDiff,
+          )}hrs to your current time`;
+        }
+      }
+
+      // Build local timezone string;
+      const hoursDiff = getHoursDiff(profileDate, currentLocalDate);
+      let printedLocalOffset = '';
+      if (hoursDiff !== null && hoursDiff !== 0) {
+        printedLocalOffset = `${decimalToHours(
+          hoursDiff,
+        )}hrs to your local time`;
+      }
+      if (
+        hoursDiff !== null &&
+        browserHoursDiff !== null &&
+        hoursDiff !== browserHoursDiff
+      ) {
+        return `${printedLocalOffset}${
+          printedBrowserOffset === '' ? '' : `, ${printedBrowserOffset}`
```

Comment:
This may result in a leading `, ` if `hoursDiff` is `null` or `0`. Which happens quite frequently if I'm looking at my profile and my system is set to a different time zone (e.g. when traveling).

Reply:
I've accommodated for this in the conditional above.